lib/jbuilder/schema/renderer.rb

```diff
@@ -25,9 +25,12 @@ def render(object = nil, title: nil, description: nil, assigns: nil, **options)
 
     options[:locals] ||= {}
     options[:locals].merge! @default_locals if @default_locals
-    options[:locals][:__jbuilder_schema_options] = { model: object&.class, title: title, description: description }
 
     @view_renderer.assign assigns if assigns
+    example = @view_renderer.render(options) # Render without schema options to extract regular Jbuilder json.
```

Comment:
Here we render with plain `Jbuilder` exactly as it happens in production, which'll give us back a string of JSON.